libs/platform/src/lib/components/thumbnail/thumbnail-image/thumbnail-image.component.ts

```diff
@@ -18,8 +18,22 @@ export class ThumbnailImageComponent {
     @Output()
     thumbnailClicked: EventEmitter<Media> = new EventEmitter();
 
+    ngOnChanges(changes: SimpleChanges): void {
+        // Select first image by default, if none has been selected
+        if (changes.mediaList && Array.isArray(this.mediaList)) {
+            const alreadySelected: boolean = this.mediaList.reduce((selected, image) => {
```

Comment:
You can use some in this case, for avoid unnecessary interactions.
`this.mediaList[0].selected = this.mediaList.some(image => image.selected);`

Reply:
OK. Change has been made!